resources/placements.json

```diff
@@ -41,8 +41,8 @@
         "apprentice den": [164, 860],
         "warrior den": [1180, 860],
         "elder den": [696, 980],
-        "leader place": [[[650, 248], "y"], [[764, 224], "y"],
-                         [[976, 232], "y"], [[806, 344], "xy"]],
```

Comment:
The missing `x` and `y`'s are intentional!  Including `x` indicates that it's okay to slightly randomize the x coord of the cat in this position.  `y` does the same thing with the y coord.  If the placement is specific enough that this randomization would make the cat look unnatural (partly off a rock or smth to that respect), then omitting the `x` or `y` gets rid of that randomized positioning.

Reply:
oh that makes a lot more sense sorry, I reverted that